docs/json_rpc_api.mediawiki

```diff
@@ -1899,6 +1903,33 @@ of the best block.
 
 ----
 
+====gettxoutsetinfo====
+{|
+!Method
+|gettxoutsetinfo
+|-
+!Parameters
+|None
+|-
+!Description
+| Returns information about an unspent transaction output.
+|-
+!Returns
+|<code>(json object)</code>
+: <code>height</code>: <code>(numeric)</code> The current block height.
+: <code>bestblock</code>: <code>(numeric)</code> The hex encoded hash of the best block.
+: <code>transactions</code>: <code>(numeric)</code> The number of unique transactions referenced by outputs.
+: <code>txouts</code>: <code>(numeric)</code> The number of transaction outputs.
+: <code>serializedhash</code>: <code>(string)</code> The merklized hash of the utxo set.
+: <code>disksize</code>: <code>(numeric)</code> The size of the utxo set on disk, in bytes.
+: <code>totalamount</code>: <code>(numeric)</code> The total value of the utxo set.
+|-
+!Example Return
+|<code>{"height": 14,"bestblock": "00000a47052f36dbed9e2224235ab974775d359eca4cf052374fabf363d57111","transactions": 14,"txouts": 43,"serializedhash": "8bcc3fe20c2b00335b3eaf42f1258149f97bda89fd643e39df0d911e4b24fc67","disksize": 0,"totalamount": 30455000000000}</code>
```

Comment:
I know this is just an example, but a disk size of zero would be an impressive feat!